skyVegasTests/depositFLowGameClient.spec.ts

```diff
@@ -0,0 +1,68 @@
+import { test, type Page } from "@playwright/test";
+import { launchGame, validateConsoleMessages, login } from "./utilities";
+import * as fs from "fs";
+
+test.describe.configure({ mode: "serial" });
+const consoleMessages: string[] = [];
+let expectedMessages;
+let page: Page;
+const games = JSON.parse(fs.readFileSync("games.json", "utf-8"));
+games.forEach((game) => {
+  test.describe(`Testing with text: ${game}`, () => {
+    test.beforeAll(async ({ browser }) => {
+      const data = fs.readFileSync("ExpectedSlotConsoleMessages.json", "utf-8");
+      expectedMessages = JSON.parse(data);
+      page = await browser.newPage();
+      await login(page);
+    });
+
+    test.afterAll(async () => {
+      await page.close();
+    });
+
+    test("Test deposit flow", async ({browser}) => {
+        page.on("console", (msg) => {
+            consoleMessages.push(msg.text());
+        });
+        await launchGame(page, game);
+        await page
+        .frameLocator('#root iframe')
+        .getByRole('button', { name: 'Deposit' })
+        .click();
+
+        validateConsoleMessages(
+            "work pls",
```

Comment:
Think I recall this was a test message from somewhere? not sure if we are able to update this to something more intuitive

Reply:
Yeah, It is redundant so I have removed it now and will push it with the next changes I make.